src/pages/home/home.ts

```diff
@@ -405,29 +408,66 @@ export class HomePage {
     });
   }
 
-  private checkUpdate(): void {
+  private checkDesktopUpdate(): void {
     this.releaseProvider
-      .getLatestAppVersion()
+      .getLatestDesktopVersion()
       .toPromise()
       .then(version => {
-        this.logger.debug('Current app version:', version);
-        var result = this.releaseProvider.checkForUpdates(version);
-        this.logger.debug('Update available:', result.updateAvailable);
-        if (result.updateAvailable) {
-          this.newRelease = true;
-          this.updateText = this.replaceParametersProvider.replace(
-            this.translate.instant(
-              'There is a new version of {{nameCase}} available'
-            ),
-            { nameCase: this.appProvider.info.nameCase }
-          );
-        }
+        this.showAvailableVersionCard(version);
       })
       .catch(err => {
         this.logger.error('Error getLatestAppVersion', err);
       });
   }
 
+  private checkMobileUpdate(): void {
+    this.releaseProvider
+      .getLatestMobileVersion()
+      .toPromise()
+      .then(
+        (response: {
+          result: { result: string };
+          latestVersions: {
+            ios: { copay: string; bitpay: string };
+            android: { copay: string; bitpay: string };
+          };
+        }) => {
+          let version: string;
+          if (this.platformProvider.isIOS) {
+            version =
+              this.appProvider.info.nameCase == 'Copay'
+                ? response.latestVersions.ios.copay
+                : response.latestVersions.ios.bitpay;
+          } else {
+            version =
+              this.appProvider.info.nameCase == 'Copay'
+                ? response.latestVersions.android.copay
+                : response.latestVersions.android.bitpay;
+          }
+          this.showAvailableVersionCard(version);
+        }
+      )
+      .catch(err => {
+        this.logger.error('Error getLatestAppVersion', err);
+      });
+  }
+
+  private showAvailableVersionCard(version) {
```

Comment:
Could you add the types of methods? `version` here is a `string` right? (Ideally we can get real types for everything, without using `any`)